nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/test-document-blob-coldstorage-contribs.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```

Comment:
Again, to simplify, this file could be named `test-coldstorage-contrib.xml` WDYT?
Also for a quicker retrieval when looking for a resource in the IDE :)